lldb/packages/Python/lldbsuite/test/lldbplatformutil.py

```diff
@@ -322,8 +342,34 @@ def getDwarfVersion():
     return "0"
 
 
+def isExpectedVersion(
+    actual_version: str, required_version: str, operator: str
+) -> bool:
+    """Returns True if actual_version matches the required_version given the operator.
+    Any operator other than the following defaults to an equality test:
```

Comment:
This default seems dangerous but it has always been this way. Maybe it's because raising an exception here gets eaten by the test framework.